static/js/components/PodcastEpisodeCard.js

```diff
@@ -20,14 +23,30 @@ type Props = {
 export default function PodcastEpisodeCard(props: Props) {
   const { episode, podcast } = props
 
+  const dispatch = useDispatch()
+  const playClick = useCallback(
+    () => {
+      dispatch(
+        setCurrentlyPlayingAudio({
+          title:       podcast.title,
+          description: episode.title,
+          url:         episode.url
+        })
+      )
+    },
+    [dispatch]
+  )
+
   return (
     <Card className="podcast-episode-card low-padding">
       <div className="left-col">
         <div className="episode-title">
           <Dotdotdot clamp={2}>{episode.title}</Dotdotdot>
         </div>
         <div className="podcast-name">{podcast.title}</div>
-        <div className="play-placeholder black-surround">Play</div>
+        <div className="play-placeholder black-surround" onClick={playClick}>
```

Comment:
we should probably rename `play-placeholder` to `play-button` while we're here :)